homeassistant/components/overkiz/select.py

```diff
@@ -51,6 +51,17 @@ def _select_option_memorized_simple_volume(
     return execute_command(OverkizCommand.SET_MEMORIZED_SIMPLE_VOLUME, option)
 
 
+def _select_option_active_zone(
+    option: str, execute_command: Callable[..., Awaitable[None]]
+) -> Awaitable[None]:
+    """Change the selected option for Active Zone(s)."""
+    # Turn alarm off when empty zone is selected
+    if option == "":
+        return execute_command(OverkizCommand.ALARM_OFF)
+
+    return execute_command(OverkizCommand.ALARM_ZONE_ON, option)
```

Comment:
Would this be better, since `option` is always a string and an empty string should be falsy. Not sure which code style core prefers.

```suggestion
    if option:
        return execute_command(OverkizCommand.ALARM_ZONE_ON, option)
    return execute_command(OverkizCommand.ALARM_OFF)
```

Reply:
I don't have a strong preference, but i like the suggestion better if i had to pick.